extensions/stackdriver/log/logger.cc

```diff
@@ -403,7 +431,9 @@ void Logger::fillHTTPRequestInLogEntry(
           request_info.duration);
   http_request->set_referer(request_info.referer);
   auto label_map = log_entry->mutable_labels();
-  (*label_map)["request_id"] = request_info.request_id;
+  if (!audit) {
+    (*label_map)["request_id"] = request_info.request_id;
```

Comment:
request id is good to have as it helps correlate problems with application logs...

Reply:
Sure we can include it then.